### DISCUSSION TITLE
Welcome to CaretTab Discussions!

### DISCUSSION THREAD

**Discussion by bluecaret** [bluecaret/carettab#320] [! News and Announcements]
Discussions will now be the place to get support and suggest new features for CaretTab. Thank you.

**Comment by Lucy2205** [comment]
In the new release of blue caret tab, it seems that time displayed at the tab at top never changes if I select a different time zone. It only keeps showing the local time zone.  I would like it show to the time zone I select at the top as it is very important for me.

**Reply by bluecaret** [reply]
I got a fix done for this. An update will go out soon, keep an eye out for the next update in a few days. Thank you.

**Comment by wzsmith** [comment]
It'd be nice to have a feature where you could save your own layouts and quickly switch between them (similar to Rainmeter). Love the new update btw!

**Reply by bluecaret** [reply]
Thank you for the suggestion, I actually already have that in mind for a near future update. Thank you!
